tests/phpunit/Mutator/BaseMutatorTestCase.php

```diff
@@ -81,19 +81,19 @@ protected function setUp(): void
             $this->fail('Input code cant be the same as mutated code');
         }
 
-        $mutants = $this->mutate($inputCode, $settings);
+        $mutatedCodes = $this->mutate($inputCode, $settings);
```

Comment:
again, the result of mutation is Mutant. As for me, the previous variable name is perfectly ok